### DISCUSSION TITLE
upos 服务器指定

### DISCUSSION THREAD

**Discussion by ljcbaby** [the1812/Bilibili-Evolved#2763] [功能建议 / Ideas]
bilibili 漫游里有类似的功能，但我感觉我电脑更需要

**Comment by the1812** [comment]
这是干啥用的

**Reply by ljcbaby** [reply]
![IMG_20211223_225132.jpg](https://user-images.githubusercontent.com/46277145/147256701-6fec1353-0ba1-4eb6-9780-d2cd65ee89cc.jpg)![Screenshot_2021-12-23-22-51-41-959_tv.danmaku.bili.jpg](https://user-images.githubusercontent.com/46277145/147256708-5e7a684c-ebf4-4f0a-ac09-994fd2117596.jpg)
